testing/endpoint.go

```diff
@@ -375,18 +375,34 @@ func (endpoint *Endpoint) SendPacket(packet exported.PacketI) error {
 // RecvPacket receives a packet on the associated endpoint.
 // The counterparty client is updated.
 func (endpoint *Endpoint) RecvPacket(packet channeltypes.Packet) error {
```

Comment:
Just out of interest, why do we even need this function now?

Reply:
It is used in all our tests. I could remove the function and update our tests, but then all our tests will discard the result anyways + external devs will need to do the same. I see no reason to remove the function since it mostly will just cause people to fix code

Reply:
Yeah, I figured. Thanks for resp